src/main/assembly/dist/cfg/application.properties

```diff
@@ -1,26 +1,39 @@
-### General settings ###
+#
+# EASY deposit-api configuration file
+#
+# Please, modify the configuration below to match your environment. Also, delete and/or modify
+# this description.
 
 #
 # Port that the HTTP-service listens on.
 #
 daemon.http.port=20190
 
 #
-# Directories for managing the deposits. The lifecycle of a deposit is:
+# Directories for managing the deposits.
 #
-# 1.   Created in the drafts directory.
-# 2.   On submit: copied to stage and then that copy is moved to the submit-to directory.
-# 3a)  If archived successfully the bag in the draft-copy is deleted
-# 3b)  If the deposit is rejected, the depositor can reopen the draft, modify it and resubmit it.
+# - stage-zips:       is used for uploads of zipped directories or complete deposits.
+# - drafts:           contains the draft deposits; single files are uploaded here directly,
+#                     unzipped directories and deposits are moved here from stage-zips.
+# - stage-for-submit: the completed draft is copied here when the user submits it.
+# - submit-to:        the deposit copy in stage-for-submit is moved here on successful completion
+#                     of a submit request.  As moving is an atomic action this prevents further
+#                     processing of incomplete deposits.
 #
-# stage and submit-to must be on a single mount to allow moving
-# stage.upload and draft must be on a single mount to allow moving
+# N.B. stage-zips and drafts must be located on the same partition in order to support atomic moving.
```

Comment:
> `stage-zips` and `drafts` must be located on the same partition in order to support atomic moving. The same holds for `stage-for-submit` and `submit-to`.

Do you know whether it is possible to verify these requirements on application startup? Not sure about it, but it seems quite important to have that tested on startup.

Reply:
created an issue for this:
https://drivenbydata.atlassian.net/browse/EASY-2001